src/opendr/perception/panoptic_segmentation/datasets/utils.py

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from typing import Optional
+from opendr.engine.data import Image as BaseImage
+
+
+class Image(BaseImage):
```

Comment:
This could be merged with the `opendr.engine.data.Image` class or renamed to `ImageWithFilename` and moved to the same engine file.

Reply:
Why do we need the filename of the image? Is this needed during training or inference?

Reply:
Refactored the code to remove the need for an additional `ImageWithFilename` implementation.